python-mini-projects/projects/Write_a_script_to_download_a_random_image_from_unsplash_and_set_it_as_wallpaper/background_linux.py

```diff
@@ -0,0 +1,27 @@
+from requests import get  # to make GET request
+from os import system, getcwd, path
+
+
+url = "https://source.unsplash.com/random"
+filename = "random.jpg"
+
+
+def download(url, file_name):
+    '''
+    downloading the file and saving it
+    '''
+    with open(file_name, "wb") as file:
+        response = get(url)
+        file.write(response.content)
+
+
+def setup(pathtofile):
+    '''
+    setting the up file
+    '''
+    system("nitrogen --set-auto {}".format(path.join(getcwd(), pathtofile)))
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This line of code makes an outdated API call to start and communicate with processes. We recommend that you use the `subprocess` module to start new processes, connect with their pipes, and get their return codes.

[Learn more](https://docs.python.org/3/library/subprocess.html)